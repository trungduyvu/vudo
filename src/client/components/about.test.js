import React from 'react'
import renderer from 'react-test-renderer'
import About from './about'

test('render', () => {
    const tree = renderer.create(<About />).toJSON()
    expect(tree).toMatchSnapshot()
})
