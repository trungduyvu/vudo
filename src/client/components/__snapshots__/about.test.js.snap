// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render 1`] = `
<div>
  About us
</div>
`;
